[v6-exp] git: options, Fix typo of SubmoduleRecursivity. Fixes #197

onee-only opened the pull request:
Fixed a typo from `SubmoduleRescursivity` to `SubmoduleRecursivity`.
It was originally pointed out in #197.

pjbgf commented:
@onee-only this is a breaking change. I changed the target branch to be `v6-exp`, can you please review the change as GitHub caused havoc in it. :sweat_smile:

onee-only commented:
> @onee-only this is a breaking change. I changed the target branch to be `v6-exp`, can you please review the change as GitHub caused havoc in it. 😅

I've updated the base commit to the latest commit on `v6-exp`

pjbgf reviewed (approved):
@onee-only thanks for resolving this. :bow:
